eclair-core/src/main/scala/fr/acinq/eclair/blockchain/watchdogs/ExplorerApi.scala

```diff
@@ -193,7 +193,7 @@ object ExplorerApi {
             val JInt(time) = block \ "timestamp"
             val JInt(bits) = block \ "bits"
             val JInt(nonce) = block \ "nonce"
-            val previousBlockHash = (block \ "previousblockhash").extractOpt[String].map(ByteVector32.fromValidHex(_).reverse).getOrElse(ByteVector32.Zeroes)
+            val previousBlockHash = (block \ "previousblockhash").extractOpt[String].map(h => BlockHash(ByteVector32.fromValidHex(h))).getOrElse(BlockHash(ByteVector32.Zeroes))
```

Comment:
Good catch, since there is no guarantee on the number of blocks returned in the test, we can't automatically check this without making the test flaky...but I fixed this in https://github.com/ACINQ/eclair/pull/2742/commits/23d44b150c8fa27ef5539dabf5688fb6aeb13aff